src/hooks/core/chat/useFetchPostNotificationHook.ts

```diff
@@ -11,7 +11,7 @@ const useFetchPostNotificationHook = () => {
     if (!localDb) return;
     const allPromises: Promise<void>[] = notifications.map((notification) => {
       const channelList = fromNotificationAPI(notification);
-      return channelList.saveIfLatest(localDb);
+      return channelList.save(localDb);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `saveIfLatest` to `save` might introduce potential issues. The `saveIfLatest` function presumably only saves the data if it's the latest, which can be useful to avoid overwriting newer data with older data in a concurrent environment. If this concurrency control is no longer needed, then this change is fine. However, if there's still a chance that older data could overwrite newer data, then this change could lead to data inconsistencies.
```diff
- return channelList.save(localDb);
+ return channelList.saveIfLatest(localDb);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ini kenapa diubah ke `save()` mas @fajarism ?

Reply:
karena ada data yang berubah tapi last updated at nya gak berubah
jadi mending di overwrite aja

Reply:
i see, noted mas